app/models/measurement.rb

```diff
@@ -5,6 +5,15 @@ class Measurement < ApplicationRecord
   WEEKDAYS = %w[sunday monday tuesday wednesday thursday friday saturday].freeze
   MIN_PRIORITY = -10_000
   MAX_PRIORITY = 10_000
+  # How many responses will at most be scheduled at once for a single protocol subscription
+  # by the RescheduleResponses job. Since this job is ran every day, as long as this number is
+  # at least as large as the number of responses per day, it will be called again before we run
+  # out of responses, so it is all fine. The only issue here is that we use the response count
+  # to give people a measure of "completion", which we now have to add a note to that says
+  # if there are more than 500 responses to be filled out by the user, this completion count
+  # or percentage can be off. (Since it just looks at how many responses were created for
+  # a protocol subscription.)
```

Comment:
aldernative would be to calculate the number of responses instead of actually scheduling them?

Reply:
The problem is that the protocol completion is requested frequently and should be fast. So what if someone schedules a protocol subscription for the next 100 years, do we really want to return a possibly unlimited size array with protocol_completion entries that takes a long time to construct (even if we're not making responses) and uses a lot of memory?

Or just say that it doesn't work right in these cases.